go/libkb/login_state.go

```diff
@@ -187,11 +187,9 @@ func (s *LoginState) Shutdown() error {
 
 		if s.loginReqs != nil {
 			close(s.loginReqs)
-			s.loginReqs = nil // block future sends to this channel
```

Comment:
to the best of my knowledge, we don't check for `nil` before trying to send on these channels, so this seems safe to me. LMK if not...